code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -158,6 +158,7 @@ update_label("John Doe", "Clowny")
 	access = list(ACCESS_MAINT_TUNNELS, ACCESS_SYNDICATE)
 	origin_tech = "syndicate=1"
 	var/anyone = FALSE //Can anyone forge the ID or just syndicate?

```

Comment:
\>has special vars deliberately designed to keep it from having hard tells, one of the only truly subtle traitor items
\>now it has a hard tell one line below it